etl/steps/data/grapher/un/2024-10-21/census_dates.meta.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - State Capacity
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  years_since_last_census:
+    variables:
+      years_since_last_census:
+        title: Years since last census
+        description_short: The number of years since the last census of this country
+        unit: years
+        short_unit: "y"
+      last_census:
```

Comment:
unlikely you'd use this so I'd probably drop this column and exclude it and instead create an indicator with [0,1] for whether there was a census in the last 10 years

Reply:
I'll add an indicator for that!